src/client/dfs/dcache.c

```diff
@@ -990,3 +991,6 @@ drec_del(dfs_dcache_t *dcache, char *path, dfs_obj_t *parent)
 
 	return dcache->drec_del_fn(dcache, path, parent);
 }
+
+// dcache_readdir(dfs_dcache_t *dcache, dfs_obj_t *obj, dfs_dir_anchor_t *anchor, struct dirent dir)
```

Comment:
Could be usefull to add some comments on why this function declaration is commented.

Reply:
since i plan to add it next :-)
note this is all going to the feature branch, so this is just the first piece of the puzzle